src/Auth.php

```diff
@@ -183,27 +185,28 @@ public function userExists($options = [])
         /** @var \DBmysql $DB */
         global $DB;
 
-        $result = $DB->request(
-            'glpi_users',
-            ['WHERE'    => $options,
-                'LEFT JOIN' => ['glpi_useremails' => ['FKEY' => ['glpi_users'      => 'id',
-                    'glpi_useremails' => 'users_id'
-                ]
-                ]
+        $result = $DB->request('glpi_users', [
```

Comment:
Maybe we should try to use the following way to do everywhere. It is a pain to have multiple different ways to pass arguments.
```suggestion
        $result = $DB->request([
            'FROM' => 'glpi_users',
```